packages/terra-application-layout/src/utils/utilityHelpers.jsx

```diff
@@ -0,0 +1,178 @@
+      userName={userData.name}
+      userDetail={userData.detail}
+      userPhoto={userData.photo}
+    />
+  );
+
+  const defaultConfig = [
+    {
+      key: defaultKeys.MENU,
+      contentLocation: UtilityUtils.LOCATIONS.BODY,
+      title: menuTitle(intl),
+      isSelectable: false,
+      isSelected: false,
+      childKeys: [
+        defaultKeys.USER_INFORMATION,
+        defaultKeys.SETTINGS,
+        defaultKeys.HELP,
+        defaultKeys.LOG_OUT,
+      ],
+    },
+    {
+      key: defaultKeys.USER_INFORMATION,
+      contentLocation: UtilityUtils.LOCATIONS.BODY,
+      title: userInformationTitle(intl),
+      content: userDataComponent,
+      isSelectable: false,
+      isSelected: false,
+      childKeys: [
+        defaultKeys.CHANGE_PHOTO,
+      ],
+    },
+    {
+      key: defaultKeys.SETTINGS,
+      contentLocation: UtilityUtils.LOCATIONS.BODY,
+      title: settingsTitle(intl),
+      isSelectable: false,
+      isSelected: false,
+      childKeys: [
+        defaultKeys.APPEARANCE,
+        defaultKeys.SECURITY,
+      ],
+    },
+    {
+      key: defaultKeys.LOG_OUT,
+      contentLocation: UtilityUtils.LOCATIONS.FOOTER,
+      title: logOutTitle(intl),
+      isSelectable: false,
+      isSelected: false,
+      childKeys: [],
+    },
+    {
+      key: defaultKeys.HELP,
+      contentLocation: UtilityUtils.LOCATIONS.BODY,
+      title: helpTitle(intl),
+      isSelectable: false,
+      isSelected: false,
+      childKeys: [
+        defaultKeys.GETTING_STARTED,
+        defaultKeys.ABOUT,
+        defaultKeys.TERMS_OF_USE,
+      ],
+    },
+    {
+      key: defaultKeys.CHANGE_PHOTO,
+      contentLocation: UtilityUtils.LOCATIONS.BODY,
+      title: changePhotoTitle(intl),
+      isSelectable: false,
+      isSelected: false,
+      childKeys: [],
+    },
+    {
+      key: defaultKeys.APPEARANCE,
+      contentLocation: UtilityUtils.LOCATIONS.BODY,
+      title: appearanceTitle(intl),
+      isSelectable: false,
+      isSelected: false,
+      childKeys: [],
+    },
+    {
+      key: defaultKeys.SECURITY,
+      contentLocation: UtilityUtils.LOCATIONS.BODY,
+      title: securityTitle(intl),
+      isSelectable: false,
+      isSelected: false,
+      childKeys: [],
+    },
+    {
+      key: defaultKeys.GETTING_STARTED,
+      contentLocation: UtilityUtils.LOCATIONS.BODY,
+      title: gettingStartedTitle(intl),
+      isSelectable: false,
+      isSelected: false,
+      childKeys: [],
+    },
+    {
+      key: defaultKeys.ABOUT,
+      contentLocation: UtilityUtils.LOCATIONS.BODY,
+      title: aboutTitle(intl),
+      isSelectable: false,
+      isSelected: false,
+      childKeys: [],
+    },
+    {
+      key: defaultKeys.TERMS_OF_USE,
+      contentLocation: UtilityUtils.LOCATIONS.BODY,
+      title: termsOfUseTitle(intl),
+      isSelectable: false,
+      isSelected: false,
+      childKeys: [],
+    },
+  ];
+
+  return reconcileChildren(defaultConfig, additionalItems);
+};
+
+export default {
```

Comment:
I am assuming there are instances where consumers want certain items of the default utility items. Would you expect them to filter the array for the specific items they want?

Reply:
That'd be their only option today. We would need to rework things quite a bit to offer a Build-a-Bear solution. This is meant to be a convenience; if it becomes inconvenient to use, they can roll their own menu entirely.